src/components/Terminal/SerialSettings/index.tsx

```diff
@@ -199,7 +203,7 @@ const SerialSettings = () => {
                         dataBitsItems,
                         serialOptions.dataBits
                     )}
-                    disabled={isConnected}
+                    disabled={settingLocked}
```

Comment:
On all of these, shouldn't they also be disabled when the serial port is connected?